src/Symfony/Component/Validator/Constraints/File.php

```diff
@@ -15,7 +15,14 @@
 use Symfony\Component\Validator\Exception\ConstraintDefinitionException;
 
 /**
+ * Validates that a value is a valid "file".
+ *
+ * A file can be one of the following:
+ *   - A string (or object with a __toString() method) path to an existing file;
+ *   - A valid {@link \Symfony\Component\HttpFoundation\File\File} object (including objects of {@link \Symfony\Component\HttpFoundation\File\UploadedFile} class).
```

Comment:
Y used inline `{@see ...}` and `{@link ...}` in the PR. Both are used in Symfony codebase. I don't know what's the appropriate syntax.

Reply:
I thought `{@link ...}` was used to reference elements, and `{@see ...}` to reference links. But the official [phpDocumentor doc](https://docs.phpdoc.org/2.9/references/phpdoc/tags/link.html) says the opposite. I'll replace with `{@see ...}` to be consistent.

Reply:
According to the phpdocumentor 3 docs, `@see` supports both while `@link` supports only URIs.